content/docs/ethernet-v0.5/fpga.md

```diff
@@ -0,0 +1,283 @@
+The system will use the Ethernet 100Base-T connection at 100 MBaud to communicate with 
+the LowRISC Linux system.
+
+## Run the pre-built FPGA demo
+
+The files you may need:
+
+ * [chip_top.bit](https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/nexys4ddr.bit):
+   The tagpipe/minion/debug enabled FPGA bitstream
+ * [boot0001.bin](https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/boot0001.bin):
+   Linux, Busybox and Berkley bootloader (BBL) packaged in one image (for NFS root filing system).
+ * [rootfs.ext2](https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/rootfs.bzip2)
+   riscv-poky root filing system ready-built for LowRISC
+
+Download and write the bitstream
+
+    cd $TOP/fpga/board/nexys4_ddr
+    curl -L https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/nexys4ddr.bit > nexys4ddr.bit
+    curl -L https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/boot0001.bin > boot0001.bin
+    curl -L https://github.com/lowRISC/lowrisc-chip/releases/download/v0.5-rc1/rootfs.bzip2 | bzip2 -d > rootfs.ext2
+    
+Convert the bitstream to quad-spi memory format
+
+    vivado -mode batch -source $TOP/fpga/common/script/cfgmem.tcl -tclargs "xc7a100t_0" nexys4ddr.bit
+
+Burn the quad-spi memory (Ensure the MODE switch is set to QSPI)
+
+    vivado -mode batch -source $TOP/fpga/common/script/program_cfgmem.tcl -tclargs "xc7a100t_0" nexys4ddr.bit.mcs
+
+## Access the boot-loader and connect to the UART console.
+
+First check the available serial ports:
+
+    ls -l /dev/*USB*
+
+which should result in a response similar to:
+
+    crw-rw---- 1 root dialout 188, 1 Apr 14 16:32 /dev/ttyUSB1
+
+In this case the device is user and group only access, so you need to be a member of the group dialout to use it.
+
+    microcom -p /dev/ttyUSB1
+
+This needs a separate terminal window as it takes over the screen. Alternatively a separate VGA display and USB keyboard may be used. You should see a display similar to the below a short delay after the PROG button is pushed.
+
+    Selftest iteration 1
+    Selftest matches=4/4, delay = 5
+    Selftest iteration 2
+    Selftest matches=8/8, delay = 5
+    Selftest iteration 3
+    Selftest matches=16/16, delay = 7
+    Selftest iteration 4
+    Selftest matches=32/32, delay = 13
+    Selftest iteration 5
+    Selftest matches=64/64, delay = 26
+    Selftest iteration 6
+    Selftest matches=128/128, delay = 52
+    Selftest iteration 7
+    Selftest matches=256/256, delay = 103
+    Selftest iteration 8
+    Selftest matches=375/375, delay = 151
+    Hello LowRISC! Thu Dec 21 18:02:20 2017
+    MAC = eee1:e2e3e4e5
+    MAC address = ee:e1:e2:e3:e4:e5.
+    IP Address:  192.168.0.51
+    Subnet Mask: 255.255.255.0
+    Enabling interrupts
+
+## Ethernet server preparation
+
+We need to make some changes on the server to enable this option. First of all create a suitable NFSroot mount point:
```

Comment:
These instructions are presumably outdated for Ubuntu 16.04 using systemd. Perhaps a simple suggestion that these steps are illustrative for a particular system (Ubuntu 14.04) and you should follow instructions for starting an NFS server on your distribution of choice?